core/src/main/java/org/apache/accumulo/core/iterators/user/IntersectingIterator.java

```diff
@@ -110,6 +111,19 @@ public TermSource(SortedKeyValueIterator<Key,Value> iter, Text term, boolean not
     public String getTermString() {
       return (this.term == null) ? "Iterator" : this.term.toString();
     }
+
+    @Override
+    public void close() throws Exception {
+      this.iter.close();
+    }
+
+    public void closeSafely() {
```

Comment:
Same as the OrIterator.. I made TermSource just implement AutoCloseable. But I could make it implement SKVI instead